.travis.yml

```diff
@@ -17,5 +17,5 @@ script:
 - test -z "$(go vet ./... |  grep -v 'refers to unknown identifier' | tee /dev/stderr)"
 - dep ensure
 - test -z "`for d in $dirs; do go build $d | grep -v 'no non-test Go files in' | tee /dev/stderr; done`"
-- docker login azureclidev.azurecr.io -u $AZURESDKDEV_ACR_SP_USERNAME -p $AZURESDK_ACR_SP_PASSWORD
+- docker login azureclidev.azurecr.io -p $AZURESDK_ACR_SP_PASSWORD -u $AZURESDKDEV_ACR_SP_USERNAME
```

Comment:
is there any way we can use certificates with travis and docker login ? kind of sucks with passwords since we need ot change them every 3 months :(

Reply:
AFAIK, docker login is only possible with username and password